GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -1188,9 +1186,8 @@ private void InitializeComponent()
             this.StashStaged.Margin = new System.Windows.Forms.Padding(0, 3, 0, 3);
             this.StashStaged.Name = "StashStaged";
             this.StashStaged.Size = new System.Drawing.Size(171, 26);
-            this.StashStaged.TabIndex = 14;
-            this.StashStaged.TabStop = false;
-            this.StashStaged.Text = "S&tash staged changes";
+            this.StashStaged.TabIndex = 107;
```

Comment:
TabIndex `106` omitted on purpose in preparation of #2260

Reply:
As well as indent _Reset author_ checkbox?

Reply:
A new panel with indent will be inserted for #2260.